addons/website_slides/models/slide_slide.py

```diff
@@ -1053,6 +1053,9 @@ def _fetch_youtube_metadata(self, image_url_only=False):
 
         self.ensure_one()
         google_app_key = self.env['website'].get_current_website().sudo().website_slide_google_app_key
+        if not google_app_key:
+            return {}, _(
+                'Please enter a Google Drive API key in your website\'s settings to have a preview.')
```

Comment:
Hello @PierreCourt 

A few comments:
- Commit message states "task-xxx", I thought that for tickets we used "opw-xxx", not entirely sure
- Please use double quotes so that you don't have to escape the message single quote
- Would use a more generic error message, as this method could be used for something else than the preview
- I don't think it's necessary to update a very old stable version of Odoo for that, it doesn't prevent anyone from working as there is a clear workaround, would probably re-target to 18.0
- When you target stable, you have to update the POT file when introducing new error messages

Let me know if you have any questions.

Thanks & cheers!

Reply:
Thanks, I will do those and re-target.